lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -618,7 +618,7 @@ void ByteCodeGenerator::InitBlockScopedContent(ParseNode *pnodeBlock, Js::Debugg
                 Js::OpCode op = (sym->GetDecl()->nop == knopConstDecl) ? Js::OpCode::InitUndeclConsoleConstFld : Js::OpCode::InitUndeclConsoleLetFld;
                 this->m_writer.ElementScopedU(op, funcInfo->FindOrAddReferencedPropertyId(propertyId));
             }
-            else
+            else if (!sym->GetIsModuleExportStorage())
```

Comment:
No, a module export can't be declared in a block scope (or anywhere except global scope). This line is for imports. When we import something, we create a const decl for it (Parser::CreateModuleImportDeclNode) and mark the symbol for that binding as symbol export storage. We get to this line when we're initializing const declarations at the top of the function. But for module imports, we don't want to initialize a root property (via InitUndeclRootConstFld) because the value is going to be loaded from module export var slots and not stored on the root object.